helpers/integration.go

```diff
@@ -554,3 +555,11 @@ func IsObjectYoungerThanThreshold(obj metav1.Object, threshold time.Duration) bo
 
 	return durationSinceObjectCreation < threshold
 }
+
+func IsPipelineRunInFinalState(detail *integrationteststatus.IntegrationTestStatusDetail) bool {
+	isFinal := detail.Status != integrationteststatus.IntegrationTestStatusTestPassed &&
+		detail.Status != integrationteststatus.IntegrationTestStatusTestFail &&
```

Comment:
Should be OR everywhere?
```suggestion
		detail.Status != integrationteststatus.IntegrationTestStatusTestFail ||
```

Reply:
I just noticed `!=` so probably function cannot be called `IsPipelineRunInFinalState` but `IsPipelineRunNOTInFinalState`